extensions/amp-story/0.1/test/test-amp-story-cta-layer.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {AmpStoryCtaLayer} from '../amp-story-cta-layer';
+
+describes.realWin('amp-story-cta-layer', {
+  amp: {
+    runtimeOn: true,
+    extensions: ['amp-story'],
+  },
+}, env => {
+  let win;
+  let ampStoryCtaLayer;
```

Comment:
Nit: could be a const in the ``beforeEach``, since you're then using ``ampStoryCtaLayer.element``?